internals/controllers/rolebinding/rolebinding_helpers.go

```diff
@@ -0,0 +1,129 @@
+package controllers
+
+import (
+	"github.com/dana-team/hns/internals/utils"
+	"github.com/go-logr/logr"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"reflect"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
+)
+
+// IsValidRoleBinding checks if a RoleBinding object is valid.
+func IsValidRoleBinding(roleBinding client.Object) bool {
```

Comment:
IsRoleBindngSnsRelated or isRoleBindingMatch because its a valid role binding everything is ok with that rolebinding he just not interesting for us.

Reply:
renamed to `isRoleBindingHNSRelated`